Explorer++/TestExplorer++/ConfigXmlStorageTest.cpp

```diff
@@ -25,3 +26,23 @@ TEST_F(ConfigXmlStorageTest, SaveLoad)
 
 	EXPECT_EQ(loadedConfig, referenceConfig);
 }
+
+TEST_F(ConfigXmlStorageTest, ConfigEnvVar)

```

Comment:
The way the tests are set up is that the tests for a particular file appear in a test file that has a matching name and ends in "Test". So, since this function appears in `Storage.cpp`, the test should go in a file named `StorageTest.cpp` in the `TestExplorer++` project.

Sorry for not making that clearer. I have written up an initial `CONTRIBUTING.md` file, where I list some of the conventions like this. I haven't committed that yet, but it should make conventions like this clearer in future.